Source/Constraint.swift

```diff
@@ -407,9 +399,7 @@ public class Constraint {
         }
         
         // store the layout constraints against the installed on view
-        var layoutConstraints = Array<LayoutConstraint>(layoutFrom!.snp_installedLayoutConstraints)
-        layoutConstraints += newLayoutConstraints
-        layoutFrom!.snp_installedLayoutConstraints = layoutConstraints
+        layoutFrom!.snp_installedLayoutConstraints += newLayoutConstraints
```

Comment:
I believe this was a hack to make beta2 work. This is already fixed in Xcode GM.